templates/common/root/_Gruntfile.js

```diff
@@ -184,13 +184,15 @@ module.exports = function (grunt) {
     },
 
     // Add vendor prefixed styles
-    autoprefixer: {
+    postcss: {
       options: {
-        browsers: ['last 1 version']
+        processors: [
+          require('autoprefixer-core')({browsers: ['last 1 version']})
```

Comment:
Sort of. This generator has always supported last 1 version. If people really want older IE support then they can change this themselves.